powershell/resources/psruntime/BuildTime/Models/PsProxyOutputs.cs

```diff
@@ -216,6 +216,25 @@ public string GetProcessCustomAttributesAtRuntime()
 {Indent}{Indent}[Microsoft.Message.ClientRuntime.MessageAttributeHelper]::ProcessPreviewMessageAttributesAtRuntime($cmdInfo, $MyInvocation, $parameterSet, $PSCmdlet)";
         }
 
+        private string GetLoginVerification()
+        {
+            if (!VariantGroup.IsInternal && IsAzure)
+            {
+                return $@"
+{Indent}{Indent}$context = Get-AzContext
+{Indent}{Indent}if (-not $context -or -not $context.Account) {{
```

Comment:
Confirm with Lei we only need to check $context here so it should be `if (-not $context)`